docs/modules/ROOT/pages/support/index.adoc

```diff
@@ -1,9 +1,13 @@
 = Support for {ProductName}
 
+
 To get support you can:
 
-* Email us - link:mailto:stonesoup@redhat.com[Stone Soup mailing list].
 
-* Contact with us in our Google Group link:https://groups.google.com/g/stonesoup[Google group: Stone soup]
+* Join us on Slack - join the link:https://dn.dev/DevNationSlack[DevNation workspace] and find us in the link:https://rhdevnation.slack.com/archives/C04LXT1EU7K[#software-supply-chain-security] channel.
```

Comment:
This is picky, but to avoid saying "join" twice I would simplify this to something like "To join us on Slack, visit {link}"

Reply:
Also avoid "-" when possible which in this case we can